buffer/src/test/java/io/netty/buffer/CompositeByteBufTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.buffer;
+
+import org.junit.Test;
+
+import static io.netty.buffer.Unpooled.*;
+import static org.junit.Assert.*;
+
+public class CompositeByteBufTest extends AdvancedLeakAwareByteBufTest {
+
+    @Test
+    public void testAddComponentWithLeakAwareByteBuf() {
```

Comment:
just merge this test into `AdvancedLeakAwareByteBufTest`. There is no need to introduce a new sub-class

Reply:
@normanmaurer Make sense. TestCase moved(https://github.com/netty/netty/pull/9398/commits/5484583121f407a086bf0a24e4d3b84c563aaa69).